app/custom/organisms/blog-list/_style.scss

```diff
@@ -0,0 +1,8 @@
+.blog-list {
```

Comment:
Is this a style specific for the list on the blog page?

If so I don't think blog-list is a accurate name or at least not so reusable, for instance what happen if I want blog-list style on a specific template?